signer/migrations/0003__create_tables.sql

```diff
@@ -184,3 +177,81 @@ CREATE TABLE sbtc_signer.withdrawal_reject_events (
     created_at    TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL
 );
 
+-- Represents an individual transaction within a broadcasted sweep transaction
+-- package. Individual deposit and withdrawal requests reference back to these
+-- transactions to keep track of both the overall transaction package as well as
+-- the individual Bitcoin transactions they are related to.
+CREATE TABLE sbtc_signer.sweep_transactions (
+    -- The Bitcoin transaction ID of the transaction.
+    txid BYTEA PRIMARY KEY NOT NULL,
+    -- The signer UTXO being spent in this transaction.
+    signer_prevout_txid BYTEA NOT NULL,
+    signer_prevout_output_index INTEGER NOT NULL,
+    signer_prevout_amount BIGINT NOT NULL,
+    signer_prevout_script_pubkey BYTEA NOT NULL,
+    -- The total amount of the transaction.
+    amount BIGINT NOT NULL,
+    -- The fee paid for the transaction.
+    fee BIGINT NOT NULL,
+    -- The Bitcoin block hash at which this package was created.
+    created_at_block_hash BYTEA NOT NULL,
+    -- The Bitcoin market fee rate at the time this package was created.
+    market_fee_rate DOUBLE PRECISION NOT NULL,
+    -- Timestamp of when this package was created.
+    created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+
+-- Represents a single withdrawal request which has been included in a sweep
+-- transaction package. Withdrawal requests have a unique ID so we use that here
+-- to reference the withdrawal request together with its Stacks block hash,
+-- which can be retrieved from the `withdrawal_requests` table.
+CREATE TABLE sbtc_signer.swept_withdrawals (
+    sweep_transaction_txid BYTEA PRIMARY KEY NOT NULL,
+    -- The index of the sweep output in the sweep transaction.
+    output_index INTEGER NOT NULL,
```

Comment:
I think this needs to be a composite primary key with these two fields.

Reply:
Yup, good catch